tests/Jellyfin.Naming.Tests/Video/CleanDateTimeTests.cs

```diff
@@ -1,143 +1,59 @@
 ﻿using System.IO;
+using Emby.Naming.Common;
+using Emby.Naming.Video;
 using Xunit;
 
 namespace Jellyfin.Naming.Tests.Video
 {
-    public class CleanDateTimeTests : BaseVideoTest
+    public sealed class CleanDateTimeTests
```

Comment:
Why did you remove the usage of the base class?

Reply:
Made it more complicated than it needs to be imo